addons/main/weapons/CfgWeapAWM.hpp

```diff
@@ -1,4 +1,5 @@
 class hlc_AWC_base: Rifle_Base_F {
+    ACE_barrelLength = 686;
```

Comment:
That's incorrect for the AWC as far as I can tell and the AWC classes already have it defined.
```suggestion
```